hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/utils/HawkbitCommonUtil.java

```diff
@@ -204,7 +188,7 @@ public static String getPollStatusToolTip(final PollStatus pollStatus, final Vaa
      * @return String formatted text
      */
     public static String getFormattedName(final String orgText) {
-        return trimAndNullIfEmpty(orgText) == null ? SPUIDefinitions.SPACE : orgText;
+        return org.apache.commons.lang3.StringUtils.trimToNull(orgText) == null ? SPUIDefinitions.SPACE : orgText;
```

Comment:
`org.apache.commons.lang3.StringUtils` commons string should be replaced by `StringUtils` from Spring.